src/main/java/org/prebid/server/validation/model/ContextSubType.java

```diff
@@ -0,0 +1,70 @@
+package org.prebid.server.validation.model;
```

Comment:
Since these enums are related to OpenRTB Native in general than to validation logic I think it's logical to move them into com.iab.openrtb.request.native package